### PR TITLE
hw11 done

### PR DESCRIPTION
**Pull request by wuyntmm**

### CODE REVIEW

File: hw11.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import time
+
+
+def name_and_time(func):
+    print(f'You called function "{func.__name__}" on {time.ctime()}')
```
Comment (klymenok) [inline hw11.py]: This log should be inside the `inner_deco_func` to work only when a decorated function is called but not when the decorator initialized

File: hw11.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    def inner_deco_func():
```
Comment (klymenok) [inline hw11.py]: Add here *args and **kwargs arguments and pass them to the `func` call to be able to decorate functions that take arguments

File: hw11.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        func()
```
Comment (klymenok) [inline hw11.py]: It is better to use `return` statement here since decorated functions can return some value

**Review by klymenok — CHANGES_REQUESTED** [review]

**Comment by wuyntmm** [comment]
done

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**